yowsup/layers/protocol_media/protocolentities/message_media_downloadable.py

```diff
@@ -20,10 +20,10 @@ class DownloadableMediaMessageProtocolEntity(MediaMessageProtocolEntity):
     '''
     def __init__(self, mediaType,
             mimeType, fileHash, url, ip, size, fileName, 
-            _id = None, _from = None, to = None, notify = None, timestamp = None, participant = None,
-            preview = None, offline = None, retry = None):
+            _id = None, _from = None, to = None, notify = None, timestamp = None, 
```

Comment:
This constructor is still being called with preview argument from the 'fromFile' static method which throws an error for example if one tries to send an image